extension/android/jni/jni_layer_llama.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include <executorch/examples/models/llama2/runner/runner.h>
 #include <executorch/examples/models/llava/runner/llava_runner.h>
+#include <executorch/examples/mediatek/executor_runner/mtk_llama_runner.h>
```

Comment:
Probably need to put this under a preprocessor macro. We don't want to add MTK header library to our build always

Reply:
aligned with @kirklandsign that he will help make change to this as he would like to refactor jni a bit.